docs-chef-io/content/inspec/resources/cloud_security.md

```diff
@@ -0,0 +1,148 @@
++++
```

Comment:
Why are there two copies of this file? There should not be one in the resources directory.

Reply:
My bad. Removed it.